stdlib/public/Observation/Sources/Observation/ObservationRegistrar.swift

```diff
@@ -53,39 +106,21 @@ public struct ObservationRegistrar<Subject: Observable>: Sendable {
     }
   }
   
-  public func willSet<Member>(
+  public func willSet<Subject: Observable, Member>(
```

Comment:
Do we need tests of `willSet` or `didSet` being used directly? Or do you consider testing the usage via `withMutation` enough?

Reply:
since the withMutation is really just a pass-through I consider that probably enough